src/platform/graphics/texture.js

```diff
@@ -201,6 +202,12 @@ class Texture {
 
         this._format = options.format ?? PIXELFORMAT_RGBA8;
         this._compressed = isCompressedPixelFormat(this._format);
+        this._integerFormat = isIntegerPixelFormat(this._format);
+        if (this._integerFormat) {
+            options.mipmaps = false;
+            options.minFilter = FILTER_NEAREST;
```

Comment:
We should handle setters of these function to ignore unsupported values as well for integer formats